./build/release/"/test/unittest" "/path/to/extension/test/*"
Filters: /path/to/extension/test/*
[0/2] (0%): /path/to/extension/test/sql/test_failure.test  ================================================================================
Wrong result in query! (/path/to/extension/test/sql/test_failure.test:8)!
================================================================================
SELECT COUNT(*) FROM read_test_results('file.json', 'pytest_json');
================================================================================
Mismatch on row 1, column count_star()(index 1)
4 <> 999
================================================================================
Expected result:
================================================================================
999
================================================================================
Actual result:
================================================================================
4

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
unittest is a Catch v2.13.7 host application.
Run with -? for options

-------------------------------------------------------------------------------
/path/to/extension/test/sql/test_failure.test
-------------------------------------------------------------------------------
/path/to/extension/duckdb/test/sqlite/test_sqllogictest.cpp:224
...............................................................................

/path/to/extension/test/sql/test_failure.test:8: FAILED:
explicitly with message:
  0

[1/2] (50%): /path/to/extension/test/sql/duck_hunt.test    [2/2] (100%): /path/to/extension/test/sql/duck_hunt.test   
===============================================================================
test cases: 2 | 1 passed | 1 failed
assertions: 4 | 3 passed | 1 failed

make: *** [extension-ci-tools/makefiles/duckdb_extension.Makefile:192: test_release_internal] Error 1